src/CMakeLists.txt

```diff
@@ -1202,8 +1202,12 @@ endif ()
 # Assuming they don't
 foreach (libs ${LINK_COMP_LIBS})
 #  set (_PKG_CONFIG_REQUIRES_PRIVATE "${_PKG_CONFIG_REQUIRES_PRIVATE} -l${libs}")
+if(TARGET ${libs})
```

Comment:
While this is a workaround - it is not the solution for the root problem.
The actual problem is that the CMake FindZLIB.cmake module does not set an OUTPUT_NAME on the target it creates, which is the second problem. The first problem is that the variable returned in ZLIB_LIBRARIES is not a target.

There will be a proper PR soon.